Python/web_scraping_Amazon

```diff
@@ -0,0 +1,76 @@
+#Amazon web scraping
+#website selected = Amazon
+
+#Scraps a page from Amazon website and collects all the product related information from the website and store them in a data frame.
+
+import pandas as pd
+import numpy as np
+import re
+from urllib.request import urlopen
+from bs4 import BeautifulSoup
+import requests
+
+
+no_pages = 1
+
+def get_data(pageNo):  
+    
+    r = requests.get('https://www.amazon.in/gp/bestsellers/dvd/21360334031/ref=zg_bs_pg_?'+str(pageNo)+'ie=UTF8&pg='+str(pageNo))
+    content = r.content
+    soup = BeautifulSoup(content)
+    
+    alls = []
+    for d in soup.findAll('div', attrs={'class':'a-section a-spacing-none aok-relative'}):
```

Comment:
If you are just specifiying the class you can use it like this:
soup.findAll('div', class_='a-section a-spacing-none aok-relative')